sql/patch.sql

```diff
@@ -47,3 +47,7 @@
 
 --  #EndIf
```

Comment:
hi @kojiromike , master doesn't use patch.sql . this should go into 7_0_4-to-7_0_5_upgrade.sql

Reply:
@bradymiller thanks, will do